examples/server.rs

```diff
@@ -14,7 +14,7 @@ fn main() {
 
     let server = Server::new(|request, mut response| {
         info!("Request received. {} {}", request.method(), request.uri());
-        Ok(response.body("Hello Rust!".as_bytes())?)
+        Ok(response.body("Hello Rust!".as_bytes().into())?)
```

Comment:
i'm not a fan of this `into()`. this is why i do not like this approach.

Reply:
(sorry if that sounded gruff, lots of activity on this right now and i really want to defend the simple interface. i appreciate your attention and care in this PR! it just might not be the direction we want to go in at the moment)

Reply:
that's okay, i understand your concern about keeping things simple. i would like to be able to use this library, but not being able to generate a response body from the handler would be a blocker for just about any project i would use this for.